mavros/src/plugins/setpoint_raw.cpp

```diff
@@ -334,12 +334,15 @@ class SetpointRawPlugin : public plugin::PluginBase,
 		tf::vectorMsgToEigen(msg->thrust, force);
 		tf::vectorMsgToEigen(msg->torque, torque);
 
-		// Transform frame ENU->NED
-		force = ftf::transform_frame_enu_ned(force);
-		torque = ftf::transform_frame_enu_ned(torque);
+		// Transform frame NWU->NED
+		force.y() = -force.y();
```

Comment:
Nice, should work. But why do all the other messages still use ENU->NED?
Should we eventually make a new function?